src/app/series/directives/series-podcast.component.css

```diff
@@ -25,3 +25,6 @@
 .feed-url button:hover, .feed-url button:focus {
   background-color: #e28b42;
 }
+input[type=checkbox] {
+  vertical-align: middle;
+}
```

Comment:
doh. fixed. Not looking forward to cleaning up all these one offs. At least we don't have to do it all at once. I'm seeing some "the styleguide provides..." PR comments in our future. :)